docs/content/design/index.md

```diff
@@ -94,6 +98,13 @@ Druid has two means of ingestion, real-time and batch. Real-time ingestion in Dr
 Batch ingestion provides exactly once guarantees and segments created via batch processing will accurately reflect the ingested data.
 One common approach to operating Druid is to have a real-time pipeline for recent insights, and a batch pipeline for the accurate copy of the data.
 
+## Querying the Data
+
+Druid's native query language is JSON over HTTP, although the community has contributed query libraries in numerous languages, including SQL.
+
+Druid is designed to perform single table operations and does not currently support joins.
+Although many production setups instrument joins at the ETL level, data must be denormalized before it is loaded into Druid.
```

Comment:
The "although" here is a bit awkward. Many production setups do joins at ETL **because** data must be denormalized before loading into Druid.